account_analytic_tag_distribution/models/account_move_line.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    def _prepare_analytic_lines(self):
+        vals = super()._prepare_analytic_lines()
```

Comment:
Here we have a conflict: what happens if there's a distribution in the standard analytic and also in the tag? I think we should disable and empty the standard analytic if we select a tag with distribution, and don't call super for that cases here.